sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserService.java

```diff
@@ -499,7 +496,8 @@ public Predicate createJurisdictionFilter(CriteriaBuilder cb, From<?, User> from
 			jurisdictionFilter = cb.equal(from.get(User.REGION), currentUser.getRegion());
 		}
 
-		return CriteriaBuilderHelper.or(cb, regionalOrNationalFilter, jurisdictionFilter);
+		Predicate seePersonalDataOutsideJurisdiction = hasRight(UserRight.SEE_PERSONAL_DATA_OUTSIDE_JURISDICTION) ? cb.conjunction() : cb.disjunction();
```

Comment:
Instead of doing this, return `cb.conjunction()` in the if block that you removed at the start of the method and leave the rest of the method untouched (avoids unnecessary code execution)